rclcpp/topics/minimal_publisher/CMakeLists.txt

```diff
@@ -26,6 +26,9 @@ ament_target_dependencies(publisher_member_function_with_type_adapter rclcpp std
 add_executable(publisher_member_function_with_unique_network_flow_endpoints member_function_with_unique_network_flow_endpoints.cpp)
 ament_target_dependencies(publisher_member_function_with_unique_network_flow_endpoints rclcpp std_msgs)
 
+add_executable(publisher_member_function_with_wait_for_all_acked member_function_with_wait_for_all_acked.cpp)
```

Comment:
can we use a shorter name for the example?

It really doesn't matter if this is using a "member function", I think we could rather use "publisher_wait_for_all_acked".